packages/drawer/src/views/DrawerView.tsx

```diff
@@ -84,6 +84,16 @@ function DrawerViewBase({
   // @ts-expect-error: the type definitions are incomplete
   useLegacyImplementation = !Reanimated.isConfigured?.(),
 }: Props) {
+  // Reanimated v3 dropped legacy v1 syntax
+  const legacyImplemenationNotAvailable =
+    require('react-native-reanimated').interpolateNode === undefined;
```

Comment:
I think there's another problem with the check, `interpolateNode` won't exist for people using Reanimated 1 package so we need to somehow check for that, maybe something like `require('react-native-reanimated').useSharedValue === undefined && require('react-native-reanimated').interpolateNode === undefined`